### PR TITLE
Update default release to 18.x, EL7 to 16.x

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Comment by evgeni** [comment]
Yes, the latest LTS is 20.x, but that had failures in CI for me and I figured let's at least update to a supported release.

File: manifests/params.pp [hunk 1]
```diff
@@ -11,7 +11,10 @@
   $repo_proxy_password         = 'absent'
   $repo_proxy_username         = 'absent'
   $repo_release                = undef
-  $repo_url_suffix             = '12.x'
+  $repo_url_suffix             = ($facts['os']['family'] == 'RedHat' and $facts['os']['release']['major'] == '7') ? {
```
Comment (bastelfreak) [inline manifests/params.pp:14]: does that render properly in puppet-strings or do you need to write it on a single line?

Comment (evgeni) [reply manifests/params.pp:14]: neither.

"Default value: `$nodejs::params::repo_url_suffix`" is what it documents today.

Comment (bastelfreak) [reply manifests/params.pp:14]: ah lol that's params.pp

**Review by bastelfreak — APPROVED** [review]

**Pull request merged**